Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -908,8 +904,6 @@ protected async Task RegisterResources(IServiceCollection serviceCollection)
             _serviceProvider = serviceCollection.BuildServiceProvider();
         }
 
-        public virtual string PackageRuntime => "netcore";
```

Comment:
Was this just not being used anywhere?

Reply:
Not anymore, this PR removes the last reference.